pkg/restore/db.go

```diff
@@ -123,13 +123,31 @@ func (db *DB) CreateTable(ctx context.Context, table *utils.Table) error {
 				utils.EncloseName(table.Db.Name.O),
 				utils.EncloseName(table.Info.Name.O))
 			if increment < 0 {
-				restoreMetaSQL += fmt.Sprintf(setValFormat, table.Info.Sequence.MinValue)
+				restoreMetaSQL = fmt.Sprintf(setValFormat, table.Info.Sequence.MinValue)
 			} else {
-				restoreMetaSQL += fmt.Sprintf(setValFormat, table.Info.Sequence.MaxValue)
+				restoreMetaSQL = fmt.Sprintf(setValFormat, table.Info.Sequence.MaxValue)
 			}
+			err = db.se.Execute(ctx, restoreMetaSQL)
+			if err != nil {
+				log.Error("restore meta sql failed",
+					zap.String("query", restoreMetaSQL),
+					zap.Stringer("db", table.Db.Name),
+					zap.Stringer("table", table.Info.Name),
+					zap.Error(err))
+				return errors.Trace(err)
+			}
+
 			// trigger cycle round > 0
-			restoreMetaSQL += nextSeqSQL
-			restoreMetaSQL += fmt.Sprintf(setValFormat, table.Info.AutoIncID)
+			err = db.se.Execute(ctx, nextSeqSQL)
+			if err != nil {
+				log.Error("restore meta sql failed",
+					zap.String("query", restoreMetaSQL),
+					zap.Stringer("db", table.Db.Name),
+					zap.Stringer("table", table.Info.Name),
+					zap.Error(err))
+				return errors.Trace(err)
+			}
+			restoreMetaSQL = fmt.Sprintf(setValFormat, table.Info.AutoIncID)
 		} else {
 			restoreMetaSQL = fmt.Sprintf(setValFormat, table.Info.AutoIncID)
 		}
```

Comment:
The `else` branch can be removed and the assignment be pulled out of the `if` statement.